### ISSUE TITLE
Entity Inspector does not save changes to child entities

### ISSUE THREAD

**Issue by KremnevDmitry** [jmix-framework/jmix#1057]
Reproduce attached: [reproduce.zip](https://github.com/jmix-framework/jmix/files/9672565/reproduce.zip)

**Comment by Flamesson** [comment]
Faced with the same problem in a project. Exclude action doesn't save result too

**Comment by Flamesson** [comment]
My analysis results:
* EditAction - doesn't save changes because of dataContext managing. I have replaced NoopDataContext to parent-child relationship and ensured dataContext.merge on setEntityToEdit. See attached snippet for details
* ExcludeAction - besides changes made in previous step I figured out that it doesn't work because of incorrect fetch plan. I deleted if-clause at InspectorFetchPlanBuilder#98 and now collections are included to fetch plan, exclude works

[edit-action.txt](https://github.com/jmix-framework/jmix/files/10472257/edit-action.txt)
